src/Data/Reader/EntityReader.php

```diff
@@ -226,4 +227,19 @@ private function resetCountCache(): void
         }
         $this->countCache = new CachedCount($newQuery);
     }
+
+    private function normalizeCriteria(array $criteria): array
```

Comment:
Please rename this method to relate it with sorting. For example:
`normalizeSorting`,  `normalizeSortCriteria`, `getSortingValues`...